src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1356,6 +1357,11 @@ def disable_features(cmd, client, resource_group_name, cluster_name, kube_config
     }
     telemetry.add_extension_event('connectedk8s', kubernetes_properties)
 
+    if disable_cluster_connect:
+        helm_values = get_all_helm_values(client, cluster_name, resource_group_name, configuration, kube_config, kube_context)
+        if not disable_cl and helm_values.get('systemDefaultValues').get('customLocations').get('enabled') == True and helm_values.get('systemDefaultValues').get('customLocations').get('oid')!="":
```

Comment:
keep this in try block, since get will return None when 'customLocations' is not there in older charts. and None.get('enabled') will throw exception.

Reply:
We will prompt warning that custom-locations enabling is not allowed in this chart version? Do confirm the prompt message once.

Reply:
wont this check be required for other features as well ?

Reply:
Doesn't matter, we have left an exposed Exception unhandled, so we have to add a try block here.

Plus, In other cases, we dont have cross/inter dependency between two features.

Reply:
Rest others features have been helm chart since GA, this custom-locations parameter is post GA.

Reply:
regardless of what we decide for the rest of the features, I have added this change for custom locations for now.